docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -670,200 +765,93 @@
+    "### N2 dissociation curve\n",
     "\n",
-    "Post-selection removes the erroneous states completely, leaving only the basis states to construct the subspace Hamiltonian. This usually saves classical resources, but might miss many states. Configuration recovery attempts to flip individual bits in erroneous basis states with respect to some probability distribution. Using configuration recovery increases the number of basis states to build a larger subspace Hamiltonian, but it uses a lot more classical resources.\n",
+    "As a performance benchmark, the 20-qubit $\\text{N}_2$ dissociation curve is shown. \n",
     "\n",
+    "![N2_dissociation](/images/guides/qunasys-qsci/N2_dissociation.png)\n",
     "\n",
-    "| Name                     | Type                        | Description                                                                                                                             | Required | Default |\n",
-    "| :----------------------- | :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- | :------- | :------ |\n",
-    "| particle_number_symmetry | bool                        | If True, remove bitstrings that do not respect particle number conservation from sampling result                                    | No       | TRUE    |\n",
-    "| spin_symmetry            | bool                        | If True, remove bitstrings that do not respect $s_z$ conservation from sampling result.                                              | No       | TRUE    |\n",
-    "| configuration_recovery   | Union[bool, dict[str, int]] | Activate configuration recovery if set to values other than False. Refer to the configuration recovery section for dict[str, int] input | No       | FALSE   |"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "a4eb76c3-c367-4555-8130-60b21fbffd87",
-   "metadata": {},
-   "source": [
-    "### Symmetry post-selection\n",
-    "\n",
-    "These keys control the symmetry post-selection: `particle_number_symmetry` and `spin_symmetry`. Setting `particle_number_symmetry` to True selects those states that conserve the total electron numbers.  Setting `spin_symmetry` to True selects those states that conserve the $s_z$ quantum number. Currently, only $s_z = 0$ is supported. The default is:"
+    "The red curve in the plot is generated by the QSCI method with the `DoubleExcitation` ansatz . You can reproduce it by using the following code with different choices of `d`, the distance between the nitrogen atoms."
```

Comment:
```suggestion
    "The red curve in the plot is generated by the QSCI method with the `DoubleExcitation` ansatz. You can reproduce it by using the following code with different choices of `d`, the distance between the nitrogen atoms."
```

Reply:
It's too late to change this now, but there should be another way of distinguishing the line for accessibility.  For example, with a dashed line.   Also, the descriptive text should have more description than "N2_dissociation", also for accessibility.